specification/resources/resource-manager/Microsoft.Authorization/2017-06-01-preview/policy.json

```diff
@@ -0,0 +1,1102 @@
+{
```

Comment:
This new spec needs to be added to the root readme.md to have it included in autorest packages, correct?

Reply:
good catch!